src/client/sandbox/node/element.js

```diff
@@ -161,9 +162,28 @@ export default class ElementSandbox extends SandboxBase {
         var overridedGetAttributeCore    = (el, attr, ns) => this._overridedGetAttributeCore(el, attr, ns);
         var overridedSetAttributeCore    = (el, attr, value, ns) => this._overridedSetAttributeCore(el, attr, value, ns);
         var overridedRemoveAttributeCore = (el, ns, arg) => this._overridedRemoveAttributeCore(el, ns, arg);
+        var overrideTextNode             = (node, parentNode) => {
```

Comment:
There is no reason to declare it here. Make it instance method. I'd prefer to move all these arrows to the instance methods BTW, I don't see any reason to have this shortcuts here.